ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-2.4.yaml

```diff
@@ -114,6 +114,28 @@ tests:
           memory: 2Gi
       timeout: 20m0s
     workflow: servicemesh-istio-e2e-aws
+- as: update-charts-in-2-4
```

Comment:
Actually I don't think we need a 2.4 job.

The whole point of these jobs is to update the latest (i.e., 2.4) charts when previous charts change. Since 2.4 is the latest, it doesn't make sense for it to exist.